docs/explanation/e-cryptography.md

```diff
@@ -0,0 +1,78 @@
+# Cryptography
+
+## Resource checksums
+
+Every version of the Charmed Kafka and Charmed ZooKeeper operators install a pinned revision of the Charmed Kafka snap
+and Charmed ZooKeeper, respectively, in order to 
+provide reproducible and secure environments. The Charmed Kafka snap and Charmed ZooKeeper snap package the 
+Kafka and ZooKeeper workload together with 
+a set of dependencies and utilities required by the lifecycle of the operators. 
+Every artifact bundled into the Charmed Kafka snap and Charmed ZooKeeper snap are verified against their checksum. 
+
+## Sources verification
+
+Charmed Kafka and Charmed ZooKeeper artifacts are published programmatically using release pipelines implemented via GitHub Actions of the 
+associated repositories. 
+All repositories containing the source code of the Charmed Kafka project (i.e. charm and snap codebases) are set up with 
+
+* Branch protection rules requiring additional commits to be merged via Pull-Request with at least 2 approvals from repository maintainers
+* Signed commits requirements (using GPG keys)
+* Requiring developers to sign CLA agreement
+
+## Encryption
+
+The Charmed Kafka operator allow to deploy a secure Kafka cluster providing encryption-in-transit capabilities out of the box 
```

Comment:
```suggestion
The Charmed Kafka operator can be deployed as a secure Kafka cluster providing encryption-in-transit capabilities out of the box 
```

`allow` is in the [Style guide's](https://docs.ubuntu.com/styleguide/en#words-and-phrases-to-avoid) avoidance list. Have you tried the new content with Vale tests? There must be some false positives there, but all results are sorted by file so it should be easy enough to find errors and suggestions for this particular file. Alternatively, you can run it against a single file. But I never tried that. Maybe I should try it someday.

Reply:
done!